db/migrate/20201001122332_add_grades_to_votes.rb

```diff
@@ -0,0 +1,5 @@
+class AddGradesToVotes < ActiveRecord::Migration[6.0]
```

Comment:
There is really no reason to create additional migrations for a single table during development. 
You've created `votes` table in this PR and you could've just added a column in the migration where you've added the table itself. 

While that's not a problem really, it really helps in a long run to keep amount of migrations in a project to a minimum.